h2o-core/src/main/java/water/api/schemas3/CloudV3.java

```diff
@@ -187,6 +189,9 @@ public NodeV3() {}
     @API(help="Memory Bandwidth", direction=API.Direction.OUTPUT)
     public double mem_bw;
 
+    @API(help="DKV", direction = API.Direction.OUTPUT)
+    public TwoDimTableV3 dkv;
```

Comment:
the cloud api is called frequently, it is not possible to include a structure of unlimited size in this response